app/src/main/java/com/example/demo/kotlin_practice/classesExamples/DemoConstructor.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.demo.kotlin_practice.classesExamples
+
+//Primary Constructor
+class DemoConstructor public constructor(var fname: String = "Patel", var age: Int) {
+
+    var firstName = fname
+    var perAge = age
```

Comment:
no need to redeclare this variables here if you are already declaring the constructer parameters as class properties